Dockerfile

```diff
@@ -27,11 +18,13 @@ RUN cd /opt/pizza-sync/frontend && yarn && yarn run build:prod && rm -rf node_mo
 RUN cd /opt/pizza-sync/backend/ && yarn && mv ../frontend/dist ./public
```

Comment:
for clarity, I think it would be best to copy frontend/dist in the last stage of the build: so that we are clear what are the input of the image : the whole backend directory and the dist frontend directory